docs(server/commands): add documentation about debugging pool sizes (#507)
citizenfx/fivem-docs: content/docs/server-manual/server-commands.md

| StaticBounds | 5000 | 6500 |
| TxdStore | 26000 | 26000 |

You can explore most of the current pools and their sizes using  `F8 > Tools > Streaming > Pool Monitor` tool.

#### Local experiments

It is possible to bypass the pool size limitations for development purposes. In this case the pool size validation will be skipped. In order to do so you need to set `moo` [convar](/docs/scripting-reference/convars/#standard-convars) to `31337` independently on server and client side:

- Add `set moo 31337` to your server config.
- Add `+set moo 31337` flag when running your client. Similarly to how it's done to turn on [the developer mode](/docs/client-manual/console-commands/#developer-commands).

> NOTE: This is for development and debugging purposes only. Never use it to bypass limitations for your production environment. If you set pool sizes outside of supported limits - you are on your own. If you believe that the limits should be adjusted - reach out to CFX team for support instead.

Reviewer: ```suggestion
{{% alert color="warning" %}}
This is for development and debugging purposes only. Never use it to bypass limitations for your production environment. If you set pool sizes outside of supported limits - you are on your own. If you believe that the limits should be adjusted - reach out to Cfx team for support instead.
{{% /alert %}}
```

This will hopefully catch peoples attentions to read this. Also changes CFX -> Cfx

You can explore current pools and their sizes using  `F8 > Tools > Streaming > Pool Monitor` tool.

#### Local experiments

> It is possible to bypass the pool size limitations for development purposes. In this case the pool size validation will be skipped. In order to do so you need to set `moo` [convar](https://docs.fivem.net/docs/scripting-reference/convars/#standard-convars) to `31337` independently on server and client side:

Reviewer 2: Nitpick, but may I suggest:

```suggestion
It is possible to bypass the pool size limitations for development purposes. In this case the pool size validation will be skipped. In order to do so you need to set `moo` [convar](/docs/scripting-reference/convars/#standard-convars) to `31337` independently on server and client side:
```

It's better not to use URLs; that way documentation won't be affected if the subdomain ever changes.

- Add `set moo 31337` to your server config.

> - Add `+set moo 31337` flag when running your client. Similarly to how it's done to turn on [the developer mode](https://docs.fivem.net/docs/client-manual/console-commands/#developer-commands).

Reviewer 2: Just nitpicking really 😊

```suggestion
- Add `+set moo 31337` flag when running your client. Similarly to how it's done to turn on [the developer mode](/docs/client-manual/console-commands/#developer-commands).
```
